docs/ref/contrib/gis/geoquerysets.txt

```diff
@@ -708,11 +708,19 @@ Backend     SQL Equivalent
 ==========  ======================================
 PostGIS     ``ST_DWithin(poly, geom, 5)``
 Oracle      ``SDO_WITHIN_DISTANCE(poly, geom, 5)``
+SpatiaLite  ``PtDistWithin(poly, geom, 5)``
 ==========  ======================================
 
 .. note::
+    The default units being queried against will vary by backend. On PostGIS
```

Comment:
```Returns models where the distance to the geometry field from the lookup
geometry are within the given distance from one another. Note that you can only
provide :class:`~django.contrib.gis.measure.Distance` objects if the targeted
geometries are in a projected system. For geographic geometries, you should use
units of the geometry field (e.g. degrees for ``WGS84``) .```

This is what is there now, I wonder if the last sentence should be amended to say that on Oracle and SpatiaLite it uses meters?